sample/src/main/res/layout/activity_main.xml

```diff
@@ -9,6 +9,25 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:text="@string/hello_world"/>
+  <TextView
+      android:id="@+id/repeated_view_1"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:visibility="gone"
```

Comment:
What do you think about adding a test that check that this TextView is invisible and the next ones visible? Just to check that the context that we're testing is the good one. If not, someone could remove this TextView, break your feature, and the tests will give a green instead of a red.